### PR TITLE
Merge release 21.05 into master 210503

### PR DESCRIPTION
**Pull request by nollymar**

### CODE REVIEW

File: .github/workflows/core-cicd-tests.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     name: Unit Tests
     runs-on: ubuntu-18.04
     env:
-      DOT_CICD_BRANCH: 19756-java-11
+      DOT_CICD_BRANCH: release-21.05
```
Comment (dsilvam) [inline .github/workflows/core-cicd-tests.yml:14]: let's put master here

File: .github/workflows/core-cicd-tests.yml [hunk 2]
```diff
@@ -74,7 +74,7 @@ jobs:
     name: Curl Tests
     runs-on: ubuntu-18.04
     env:
-      DOT_CICD_BRANCH: 19756-java-11
+      DOT_CICD_BRANCH: release-21.05
```
Comment (dsilvam) [inline .github/workflows/core-cicd-tests.yml:77]: and here

File: .github/workflows/core-cicd-tests.yml [hunk 3]
```diff
@@ -159,7 +159,7 @@ jobs:
       matrix:
         databaseType: [ postgres, mysql, oracle, mssql ]
     env:
-      DOT_CICD_BRANCH: 19756-java-11
+      DOT_CICD_BRANCH: release-21.05
```
Comment (dsilvam) [inline .github/workflows/core-cicd-tests.yml:162]: and here

File: .github/workflows/release-process.yml [hunk 4]
```diff
@@ -7,7 +7,7 @@ jobs:
     name: Release Process Automation
     runs-on: ubuntu-18.04
     env:
-      DOT_CICD_BRANCH: master
+      DOT_CICD_BRANCH: release-21.05
```
Comment (dsilvam) [inline .github/workflows/release-process.yml:10]: and here

File: .github/workflows/core-cicd-tests.yml (hunk 1 above)
Comment (nollymar) [reply .github/workflows/core-cicd-tests.yml]: done

File: .github/workflows/core-cicd-tests.yml (hunk 2 above)
Comment (nollymar) [reply .github/workflows/core-cicd-tests.yml]: done

File: .github/workflows/core-cicd-tests.yml (hunk 3 above)
Comment (nollymar) [reply .github/workflows/core-cicd-tests.yml]: done

File: .github/workflows/release-process.yml (hunk 4 above)
Comment (nollymar) [reply .github/workflows/release-process.yml]: done

**Review by dsilvam — APPROVED** [review]

**Review by jdotcms — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

**Pull request merged**